### ISSUE TITLE
Getting binance ticksize value through unified API possible?

### ISSUE THREAD

**Issue by klover1** [ccxt/ccxt#4325]
- OS: ubuntu
- Programming Language version: python
- CCXT version: 1.18.30
- Exchange: binance
- Method:

Hi, 
I've had a look at the manual and various issues here but I am unable to work out how to get the quote minimum 'values' from the unified API markets ( ['precision'] and ['limits'] ) for calculating stoplosses and so on. 

For example, for ETH/USDT the minimum increment amount is 0.01 USDT and I can see that through the exchange specific ['PRICE_FILTER']['ticksize'].

Is there no way to calculate this from one of the unified API keys? 

Thanks. 

```
{'fee_loaded': False,
 'percentage': True,
 'tierBased': False,
 'taker': 0.001,
 'maker': 0.001,
 'precision': {'base': 8, 'quote': 8, 'amount': 5, 'price': 8},
 'limits': {'amount': {'min': 1e-05, 'max': 10000000.0},
  'price': {'min': None, 'max': None},
  'cost': {'min': 10.0, 'max': None}},
 'id': 'ETHUSDT',
 'symbol': 'ETH/USDT',
 'base': 'ETH',
 'quote': 'USDT',
 'baseId': 'ETH',
 'quoteId': 'USDT',
 'info': {'symbol': 'ETHUSDT',
  'status': 'TRADING',
  'baseAsset': 'ETH',
  'baseAssetPrecision': 8,
  'quoteAsset': 'USDT',
  'quotePrecision': 8,
  'orderTypes': ['LIMIT',
   'LIMIT_MAKER',
   'MARKET',
   'STOP_LOSS_LIMIT',
   'TAKE_PROFIT_LIMIT'],
  'icebergAllowed': True,
  'filters': [{'filterType': 'PRICE_FILTER',
    'minPrice': '0.00000000',
    'maxPrice': '0.00000000',
    'tickSize': '0.01000000'},
   {'filterType': 'PERCENT_PRICE',
    'multiplierUp': '10',
    'multiplierDown': '0.1',
    'avgPriceMins': 5},
   {'filterType': 'LOT_SIZE',
    'minQty': '0.00001000',
    'maxQty': '10000000.00000000',
    'stepSize': '0.00001000'},
   {'filterType': 'MIN_NOTIONAL',
    'minNotional': '10.00000000',
    'applyToMarket': True,
    'avgPriceMins': 5},
   {'filterType': 'ICEBERG_PARTS', 'limit': 10},
   {'filterType': 'MAX_NUM_ALGO_ORDERS', 'maxNumAlgoOrders': 5}]},
 'active': True}
```

**Comment by Pmant** [comment]
On Binance there is a mininum cost, no minimum amount. You can get that form 
`limits.cost.min`
Price precision is broken right now, see #4326. For **ETH/USDT** it should be **2**, derived from **tickSize**.

**Comment by kroitor** [comment]
@klover1 @Pmant thx for reporting, the price precision was fixed in https://github.com/ccxt/ccxt/commit/dc3bd40b4c6e24fc23211b1aba29df8a5ac89947

Duplicates:
- https://github.com/ccxt/ccxt/issues/4326 (fixed in 1.18.31, will be available in 5-10 minutes)
- https://github.com/ccxt/ccxt/issues/4286

**Comment by kroitor** [comment]
@klover1 thx to you for the feedback as well! ) Let us know if anything.
